scripts/ibc_relay_tester.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+DEST_ADDR=${DEST_ADDR:-""}
+ROLLAPP_ID=${ROLLAPP_ID:-""}
+ROLLER_CONFIG_PATH=${ROLLER_CONFIG_PATH:-""}
+
+if [ -z "$DEST_ADDR" ] || [ -z "$ROLLAPP_ID" ] || [ -z "$ROLLER_CONFIG_PATH" ]; then
+    echo "Error: One or more required environment variables (DEST_ADDR, ROLLAPP_ID, ROLLER_CONFIG_PATH) are unset."
+    exit 1
+fi
+
+TIMEOUT=1800 # 30 minutes
+
+check_balance() {
+    BALANCE=$(/usr/local/bin/dymd query bank balances $DEST_ADDR | grep "ibc/")
```

Comment:
what about --home for the dymd?

Reply:
You don't need it for querying the balance

Reply:
so u need --node?